### ISSUE TITLE
Support Athena querying for CloudTrail data

### ISSUE THREAD

**Issue by mcpeak** [Netflix/repokid#93]
Athena can be used to query the S3 buckets where CloudTrail data is stored directly. This would be useful for organizations without some other storage (such as ElasticSearch) set up. It should be reasonably easy to develop a hook that makes Athena queries to determine which actions have been used for a role in the last n days.

**Comment by mcpeak** [comment]
Awesome!! Please join us in our Gitter channel: https://gitter.im/netflix-repokid/Lobby . I look forward to speaking with you!

**Comment by cByst** [comment]
@mcpeak It is my understanding that currently aardvark is utilizing access advisor to determine which services a iam role hasn't used in some time. Is the idea here that with trail logs it would allow repokid to look more granularly at this and go down to which permission in a given role that hasn't been used in some time rather than just the high service?

**Comment by mcpeak** [comment]
@cByst yes, exactly. So even if parts of a service have been used we can take away the actions within the service that haven't been.

**Comment by nasehim7** [comment]
Hi @cByst. Hope you are doing great. Yes, we have started working on it. Soon it will be in place and Sure, when we will need help, we will call out for you. :D

**Comment by zboni-gpsw** [comment]
Hi folks, what's the status of this enhancement? It's been 9 months since the last comment about WIP... thanks.
